src/ts/editor/field/media.ts

```diff
@@ -306,7 +306,7 @@ export class MediaField
       return value.url;
     }
 
-    if (value && value.path) {
+    if (value && value.path && typeof value.path === 'string') {
```

Comment:
I've been using `DataType.isString()` for checking data types. Can you update this to also use the `DataType` class?